sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/ChatRequestAssistantMessage.java

```diff
@@ -220,6 +237,19 @@ public ChatRequestAssistantMessage(BinaryData content) {
      * @param content the String content value to set.
      */
     public ChatRequestAssistantMessage(String content) {
-        this.content = BinaryData.fromString(content);
+        this.content = content == null ? null : BinaryData.fromString(content);
+        this.stringContent = content;
+        this.chatMessageContentItem = null;
```

Comment:
Should we do `this(BinaryData.fromString(content))` so we get branch coverage for the generated constructor as well?

Reply:
We can't. As the other `final` properties can't not be assigned.